DFC.Composite.Shell/Views/Shared/_ValidationScriptsPartial.cshtml

```diff
@@ -1,18 +1,20 @@
 <environment include="Development">
-    <script src="~/lib/jquery-validation/jquery.validate.js"></script>
-    <script src="~/lib/jquery-validation-unobtrusive/jquery.validate.unobtrusive.js"></script>
+    <script src="~/lib/jquery-validation/jquery.validate.js" nws-csp-add-nonce="true"></script>
+    <script src="~/lib/jquery-validation-unobtrusive/jquery.validate.unobtrusive.js" nws-csp-add-nonce="true"></script>
 </environment>
 <environment exclude="Development">
     <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-validate/1.19.0/jquery.validate.js"
             asp-fallback-src="~/lib/jquery-validation/jquery.validate.min.js"
             asp-fallback-test="window.jQuery && window.jQuery.validator"
             crossorigin="anonymous"
-            integrity="sha256-vhi8Tw3fBc+L8T6WvxZ/rmdB0AwBqVDtxc8rkK/Vuhc=">
+            integrity="sha256-vhi8Tw3fBc+L8T6WvxZ/rmdB0AwBqVDtxc8rkK/Vuhc="
+            nws-csp-add-nonce="true">
     </script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-validation-unobtrusive/3.2.11/jquery.validate.unobtrusive.min.js"
             asp-fallback-src="~/lib/jquery-validation-unobtrusive/jquery.validate.unobtrusive.min.js"
             asp-fallback-test="window.jQuery && window.jQuery.validator && window.jQuery.validator.unobtrusive"
             crossorigin="anonymous"
-            integrity="sha256-9GycpJnliUjJDVDqP0UEu/bsm9U+3dnQUH8+3W10vkY=">
+            integrity="sha256-9GycpJnliUjJDVDqP0UEu/bsm9U+3dnQUH8+3W10vkY="
+            nws-csp-add-nonce="true">
     </script>
```

Comment:
Should be using CDN, can we remove this pls?

Reply:
Removed